### DISCUSSION TITLE
CORS issue

### DISCUSSION THREAD

**Discussion by tva77** [nocodb/nocodb#340] [Q&A]
is there any way to deal with CORS issue when calling the API from a web app?

**Comment by pranavxc** [comment]
If you are creating an app using node then you can use the npm `cors` package. With docker, you may have to do some proxy.

https://www.npmjs.com/package/cors#configuring-cors

**Comment by tva77** [comment]
I'm using Flutter Web to request the API from nocodb hosted at heroku. Is there any config on nocodb to allow CORS?
